src/AdminConsole/AdminConsole.csproj

```diff
@@ -55,5 +55,12 @@
   <Target Name="Run NPM Build" AfterTargets="PostBuildEvent">
     <Exec Command="npm install" />
     <Exec Command="npm run build" />
+    
+    <!-- ES Module Shims -->
+    <Copy SourceFiles="node_modules\es-module-shims\dist\es-module-shims.js" DestinationFolder="wwwroot\lib\es-module-shims\" />
+    
+    <!-- Vue.js -->
+    <Copy SourceFiles="node_modules\vue\dist\vue.esm-browser.js" DestinationFolder="wwwroot\lib\vue\" />
+    <Copy SourceFiles="node_modules\vue\dist\vue.esm-browser.prod.js" DestinationFolder="wwwroot\lib\vue\" />

```

Comment:
Maybe because of this? https://learn.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-8.0&tabs=visual-studio#create-an-rcl-with-static-assets

It still doesn't inject them automatically unfortunately...